packages/salesforcedx-vscode-core/package.json

```diff
@@ -194,15 +194,15 @@
       "editor/context": [
         {
           "command": "sf.retrieve.current.source.file",
-          "when": "sf:project_opened && resourceLangId != 'forcesourcemanifest' && sf:has_target_org"
+          "when": "sf:project_opened && resourceLangId != 'forcesourcemanifest' && resourceLangId != 'yaml' && sf:has_target_org"
```

Comment:
From my understanding, you need to define the resourceLangId yaml [here](https://github.com/forcedotcom/salesforcedx-vscode/blob/feat/apex-oas/packages/salesforcedx-vscode-core/package.json#L1005) in order to use it.

Reply:
I believe it's not needed. I tested it just like this and works fine

Reply:
Thanks for the confirmation!